### PR TITLE
pull-request-template

### LINKED ISSUE #25: Create templates for issues and PRs
## Issue Overview

Currently there is no templates for new issues or new PRs

## Proposed Solutions or Ideas

Create this two templates, look at this [reference](https://github.com/Scaffold-Stark/scaffold-stark-2/tree/main/.github)

### PR DESCRIPTION
**Pull request by zleypner**
Hello @gianalarcon 
I hope this message finds you well. I’m currently working on the issue and will continue later. Please let me know if there’s any specific information you’d like me to add or remove from the solution.

Thank you for your guidance!

This PR closes #25

### CODE REVIEW

**Comment by gianalarcon** [comment]
Hi @zleypner. Overall looks great
We could make the PR template a bit more concise, and some section can be optional like section 5 and 6

**Comment by zleypner** [comment]
Thank you for the feedback, I will update it asap.

**Comment by zleypner** [comment]
Hello @gianalarcon, I just updated the PR template as requested. Proceeding with the issues template

**Comment by zleypner** [comment]
I just added the issues template.

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Issue | Checkers
+
+<p align="center">
```
Comment (gianalarcon) [inline .github/issue_template.md]: lets remove the images for both files, then we are ready to merge

**Comment by zleypner** [comment]
Hello @gianalarcon , Please let me know if you an additional request.

File: .github/issue_template.md (hunk 1 above)
Comment (gianalarcon) [reply .github/issue_template.md]: > let's remove the images for both files, then we are ready to merge

here @zleypner. Sorry I forgot to submit review

**Comment by zleypner** [comment]
Hello @gianalarcon ! I just removed the images.

**Review by gianalarcon — APPROVED** [review]

**Comment by gianalarcon** [comment]
Thanks for your contribution!

> Hello @gianalarcon ! I just removed the images.

**Pull request merged**